examples/run/cuda/seq_example_cuda.cpp

```diff
@@ -98,142 +89,117 @@ int seq_run(const traccc::full_tracking_input_config& i_cfg,
         sd_performance_writer.add_cache("CUDA");
     }
 
+    traccc::performance::timing_info elapsedTimes;
+
     // Loop over events
     for (unsigned int event = common_opts.skip;
          event < common_opts.events + common_opts.skip; ++event) {
 
-        /*time*/ auto start_wall_time = std::chrono::system_clock::now();
-
-        /*time*/ auto start_file_reading_cpu = std::chrono::system_clock::now();
-
-        // Read the cells from the relevant event file into host memory.
-        const traccc::cell_container_types::host cells_per_event =
-            traccc::read_cells_from_event(event, common_opts.input_directory,
-                                          common_opts.input_data_format,
-                                          surface_transforms, digi_cfg,
-                                          host_mr);
-
-        /*time*/ auto end_file_reading_cpu = std::chrono::system_clock::now();
-        /*time*/ std::chrono::duration<double> time_file_reading_cpu =
-            end_file_reading_cpu - start_file_reading_cpu;
-        /*time*/ file_reading_cpu += time_file_reading_cpu.count();
-
-        /*-----------------------------
-              Clusterization and Spacepoint Creation (cuda)
-          -----------------------------*/
-        /*time*/ auto start_cluterization_cuda =
-            std::chrono::system_clock::now();
-
-        // Copy the cell data to the device.
-        const traccc::cell_container_types::buffer cells_cuda_buffer =
-            cell_h2d(traccc::get_data(cells_per_event));
-
-        // Reconstruct it into spacepoints on the device.
-        const traccc::spacepoint_container_types::buffer
-            spacepoints_cuda_buffer = ca_cuda(cells_cuda_buffer);
-
-        /*time*/ auto end_cluterization_cuda = std::chrono::system_clock::now();
-        /*time*/ std::chrono::duration<double> time_clusterization_cuda =
-            end_cluterization_cuda - start_cluterization_cuda;
-        /*time*/ clusterization_sp_cuda += time_clusterization_cuda.count();
-
+        // Instantiate host containers/collections
+        traccc::cell_container_types::host cells_per_event;
         traccc::clusterization_algorithm::output_type measurements_per_event;
         traccc::spacepoint_formation::output_type spacepoints_per_event;
+        traccc::seeding_algorithm::output_type seeds;
+        traccc::track_params_estimation::output_type params;
 
-        if (run_cpu) {
+        // Instantiate cuda containers/collections
+        traccc::spacepoint_container_types::buffer spacepoints_cuda_buffer{
```

Comment:
@krasznaa in line with your suggestion in the other PR, should I change all these to ouput types? I'm not sure I like it much because the constructors being called aren't exactly "generic" themselves, and it seems slightly more readable like this.

Reply:
Fair question... :thinking: Let's leave them with this type for now. Once https://github.com/acts-project/vecmem/pull/208 arrives in this project, we will switch to creating the buffers with their default constructors. At that point these examples should switch to using `output_type` types. Since they'll be default-constructing the objects themselves.

But for now this naming/formalism is fine with me.